cocos/platform/apple/CCFileUtils-apple.mm

```diff
@@ -60,7 +60,8 @@ static void addItemToArray(id item, ValueVector& array)
         const char* numType = [num objCType];
         if(num == (void*)kCFBooleanFalse || num == (void*)kCFBooleanTrue)
         {
-            array.push_back(Value([num boolValue]));
+            bool v = [num boolValue];
+            array.push_back(Value(v));
```

Comment:
`Value([num boolValue])` will call 
`Value::Value(int v)
: _type(Type::INTEGER)
{
    _field.intVal = v;
}
`
this is wrong calling, the correct calling should be
`Value::Value(bool v)
: _type(Type::BOOLEAN)
{
    _field.boolVal = v;
}`
so add `bool v = [num boolValue];`

wrong:
![wrong](https://cloud.githubusercontent.com/assets/2960368/13097498/84dc0e40-d55c-11e5-8dad-954e019b60e7.png)

correct:
![correct](https://cloud.githubusercontent.com/assets/2960368/13097505/9157d154-d55c-11e5-912b-f628f1cf8600.png)